src/repositories/StockCountRepository.js

```diff
@@ -0,0 +1,10 @@
+const service = 'SC.asmx/'
+
+export const StockCountRepository = {
```

Comment:
already we have stockCount repository, please remove this one and move endpoints to existing one